jekyll/_cci2/server/installation/installation-reference.adoc

```diff
@@ -735,32 +747,44 @@ a|
 ----
 {"bucketName":"","expireAfter":0,"gcs":{"enabled":false,"service_account":{"project_id":"... ...","type":"service_account"},"workloadIdentity":""},"s3":{"accessKey":"","enabled":false,"endpoint":"https://s3.us-east-1.amazonaws.com","irsaRole":"","secretKey":""}}
 ----
-|Object storage for build artifacts, audit logs, test results and more. One of object_storage.s3.enabled or object_storage.gcs.enabled must be true for the chart to function.
+|Object storage for build artifacts, audit logs, test results and more. One of 'object_storage.s3.enabled' or 'object_storage.gcs.enabled' must be true for the chart to function.
 
 |object_storage.expireAfter
 |int
 |`0`
-|number of days after which artifacts will expire
+|number of days after which artifacts will expire.
 
 |object_storage.gcs.service_account
 |object
 |`{"project_id":"... ...","type":"service_account"}`
-|GCP Storage (GCS) Authentication Config (3 Options). Option 1: Set service_account with the service account JSON (raw JSON, not a string), and CircleCI will create the secret for you. Option 2: Leave the service_account field as its default, and create the secret yourself. CircleCI will assume it exists. Option 3: Leave the service_account field as its default, and set the workloadIdentity field with a service account email to use workload identities.
+a|GCP Storage (GCS) authentication config. 
+
+**Option 1:** Set service_account with the service account JSON (raw JSON, not a string), and CircleCI will create the secret for you. 
+
+**Option 2:** Leave the service_account field as its default, and create the secret yourself. CircleCI will assume it exists. 
+
+**Option 3:** Leave the `service_account` field as its default, and set the workloadIdentity field with a service account email to use workload identities.
 
 |object_storage.s3
 |object
 |`{"accessKey":"","enabled":false,"endpoint":"https://s3.us-east-1.amazonaws.com","irsaRole":"","secretKey":""}`
-|S3 Configuration for Object Storage. Authentication methods: AWS Access/Secret Key, and IRSA Role
+|S3 Configuration for Object Storage. Authentication methods: AWS Access/Secret Key, and IRSA Role.
 
 |object_storage.s3.accessKey
 |string
 |`""`
-|AWS Authentication Config (3 Options). Option 1: Set accessKey and secretKey here, and CircleCI will create the secret for you. Option 2: Leave accessKey and secretKey blank, and create the secret yourself. CircleCI will assume it exists. Option 3: Leave accessKey and secretKey blank, and set the irsaRole field (IAM roles for service accounts), also set region: "your-aws-region".
+a|AWS Authentication Config. 
+
+**Option 1:** Set accessKey and secretKey here, and CircleCI will create the secret for you. 
+
+**Option 2:** Leave accessKey and secretKey blank, and create the secret yourself. CircleCI will assume it exists. 
+
+**Option 3:** Leave accessKey and secretKey blank, and set the irsaRole field (IAM roles for service accounts), also set region: "your-aws-region".
```

Comment:
Option 3 is a little confusing. Is the irsaRole directly related to the access and secret keys? The "...and set the..." makes it seem like the two are connected, but then it's ALSO asking to set the region. Can it just be

> Leave accessKey and secretKey blank, set the irsaRole field (IAM roles for service accounts), and set region: "your-aws-region".

Should it be "your-aws-region" or `your-aws-region`?